modules/src/address.rs

```diff
@@ -0,0 +1 @@
+pub struct AccAddress(Vec<u8>;
```

Comment:
Yes, we need to make this change. Didn't do it here because it touches all messages. We don't do anything with this field in `Msg` in IBC modules. It is going to be handled up the stack once we have auth, accounts, etc. Can you open an issue for this?